README.md

```diff
@@ -10,6 +10,14 @@ cauliflower-filter
 new rebuild() api, ensuring stable outputs across builds. Also offers test
 coverage to ensure that this plugin behaves as expected.
 
+Cauliflower-filter aims to be 100% API compatible and suitable as a drop-in
+replacement of broccoli-filter.
+
+Key Differences                   | cauliflower-filter | broccoli-filter
+--------------------------------- | ------------------ | ---------------
+Output path stable across builds  | ✓                  | ✘
+Test coverage                     | ✓                  | ✘
```

Comment:
I have no opinions on which of those tests to pull to be honest. Perhaps I should pull @caitp's test suite, since she's actively working on Filter?

As for the "keeping output paths stable" thing... I'm not super-opposed, but it seems like an orthogonal feature that belongs in broccoli-plugin and/or the next Broccoli version.

> hopefully this whole repo can just become broccoli-filter

I'm inclined to do that. @caitp, any opinion? Should I review & merge your master branch?